vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/Exhibition.java

```diff
@@ -148,6 +148,15 @@ public void setPreviewId(String previewId) {
         this.previewId = previewId;
     }
     
+    /**
+     * 
+     * Returns the default language of the given exhibition
+     */
+    @Override
+    public IExhibitionLanguage getDefaultLanguage() {
+        return  this.getLanguages().stream().filter(language -> language.isDefault()).findFirst().orElse(null);
```

Comment:
this still

Reply:
This is an Entity class, using a service or a repository to make a db call is considered as bad practice( online resources). Should I still make a db call here?

Reply:
Can you provide the resource? The way it is written right now, iterates over all language objects to find a specific one rather than making a call for the one object to the db directly. It might be a bad practices to make a call to get all languages instead of getting the language objects from the exhibition objects, but I can't see how this would be better than just asking for the one object we want directly.

Reply:
?